projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -171,6 +155,19 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
     @ViewChild(IgxForOfDirective, { read: IgxForOfDirective, static: true })
     protected virtDir: IgxForOfDirective<any>;
 
+    @Input()
+    public set overlaySettings(val: OverlaySettings) {
+        this._overlaySettings = Object.assign({}, this._overlaySettings, val);
```

Comment:
this will not work for PositionStrategy. You should merge the PositionStrategies and reassign them.